src/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
```

Comment:
lets avoid this pattern. I'd rather we not export every theme, reducer, helper, component... etc from one major index file. This has issues with importing one thing loads everything.